tests/objc-cli/libmanaged/Tests/Tests.m

```diff
@@ -693,60 +693,63 @@ - (void)testProtocols {
-    XCTAssertTrue ([[Methods_AllOperators bitwiseOr:oneAll c2:twoAll] value] == (1 | 2), "1 | 2 All");
-    XCTAssertTrue ([[Methods_AllOperators exclusiveOr:oneAll c2:twoAll] value] == (1 ^ 2), "1 ^ 2 All");
-
-    XCTAssertTrue ([[Methods_AllOperators leftShift:oneAll a:2] value] == (1 << 2), "1 << 2 All");
-    XCTAssertTrue ([[Methods_AllOperators rightShift:oneAll a:2] value] == (1 >> 2), "1 >> 2 All");
-
-    XCTAssertTrue ([[Methods_AllOperators onesComplement:oneAll] value] == (~1), "!1 All");
-    XCTAssertTrue ([[Methods_AllOperators unaryNegation:oneAll] value] == (-1), "- 1 All");
-    XCTAssertTrue ([[Methods_AllOperators unaryPlus:oneAll] value] == (+1), "+ 1 All");
-    
-    XCTAssertTrue ([[Methods_AllOperators decrement:oneAll] value] == (1 - 1), "1 - 1 All");
-    XCTAssertTrue ([[Methods_AllOperators increment:oneAll] value] == (1 + 1), "1 + 1 All");
-
-    
-    // Methods_AllOperatorsWithFriendly defines both the operator and "friendly" version
-    Methods_AllOperatorsWithFriendly * oneFriend = [[Methods_AllOperatorsWithFriendly alloc] initWithVal:1];
-    Methods_AllOperatorsWithFriendly * twoFriend = [[Methods_AllOperatorsWithFriendly alloc] initWithVal:2];
-    XCTAssertTrue ([[Methods_AllOperatorsWithFriendly addC1:oneFriend c2:twoFriend] value] == (1 + 2), "1 + 2 All");
-    XCTAssertTrue ([[Methods_AllOperatorsWithFriendly subtractC1:oneFriend c2:twoFriend] value] == (1 - 2), "1 - 2 All");
-    XCTAssertTrue ([[Methods_AllOperatorsWithFriendly multiplyC1:oneFriend c2:twoFriend] value] == (1 * 2), "1 * 2 All");
-    XCTAssertTrue ([[Methods_AllOperatorsWithFriendly divideC1:twoFriend c2:oneFriend] value] == (2 / 1), "2 / 1 All");
-    
-    XCTAssertTrue ([[Methods_AllOperatorsWithFriendly bitwiseAndC1:oneFriend c2:twoFriend] value] == (1 & 2), "1 & 2 All");
-    XCTAssertTrue ([[Methods_AllOperatorsWithFriendly bitwiseOrC1:oneFriend c2:twoFriend] value] == (1 | 2), "1 | 2 All");
-    XCTAssertTrue ([[Methods_AllOperatorsWithFriendly xorC1:oneFriend c2:twoFriend] value] == (1 ^ 2), "1 ^ 2 All");
-    
-    XCTAssertTrue ([[Methods_AllOperatorsWithFriendly leftShiftC1:oneFriend a:2] value] == (1 << 2), "1 << 2 All");
-    XCTAssertTrue ([[Methods_AllOperatorsWithFriendly rightShiftC1:oneFriend a:2] value] == (1 >> 2), "1 >> 2 All");
-    
-    XCTAssertTrue ([[Methods_AllOperatorsWithFriendly onesComplementC1:oneFriend] value] == (~1), "!1 All");
-    XCTAssertTrue ([[Methods_AllOperatorsWithFriendly negateC1:oneFriend] value] == (-1), "- 1 All");
-    XCTAssertTrue ([[Methods_AllOperatorsWithFriendly plusC1:oneFriend] value] == (+1), "+ 1 All");
-    
-    XCTAssertTrue ([[Methods_AllOperatorsWithFriendly decrementC1:oneFriend] value] == (1 - 1), "1 - 1 All");
-    XCTAssertTrue ([[Methods_AllOperatorsWithFriendly incrementC1:oneFriend] value] == (1 + 1), "1 + 1 All");    
+    Overloads_OperatorCollision * one = [[Overloads_OperatorCollision alloc] initWithVal:1];
```

Comment:
without going back (in this PR) to existing tests please make sure all new ones (and updates) are using tabs (not spaces)